broadcast/src/main/java/org/elasticsoftware/elasticactors/broadcast/state/BroadcasterState.java

```diff
@@ -21,7 +30,6 @@
     private final List<ActorRef> nodes;
     private final Set<ActorRef> leaves;
     private boolean leafNode = true;
-    private ThrottleConfig throttleConfig;
```

Comment:
you removed backwards compatibiliy. this means that once you upgrade there is no way back. It is probably a good idea to keep at least the state compatible

Reply:
Deserialization is unnaffected, and rolling back is the same as having the actor state with ThrottleConfig as null once again. See https://github.com/elasticsoftwarefoundation/elasticactors-systems/blob/f7441564fbe44f3a5c1d17af9fbd58cf2940ca06/broadcast/src/main/java/org/elasticsoftware/elasticactors/broadcast/state/BroadcasterState.java#L26

Reply:
ok, but the orignal config does get lost, so this can be a bit unexpected

Reply:
I modified it so it won't get lost anymore